samwise.js

```diff
@@ -3,11 +3,11 @@
  *
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.